UnitedIncome/components
# ItemizationBox Animation Changes

### Description
> Please provide a brief description of your changes below.

Changes the `ItemizationBox` animations so the button link is _always_ at the bottom. I've had to include a fadein/fadeout animation for the button in order to achieve this otherwise it will appear the second the drawer opens. This resolves the issue in the dashboard where the inputs are misaligned if there's more items in one box than the other.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Link the component library with `yarn link` and test this on the dashboard. It should look much cleaner.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

![Screen Shot 2019-12-05 at 12 11 22 PM](https://user-images.githubusercontent.com/10888441/70257624-a02bb000-1758-11ea-9e8a-d8a7ea9b76e3.png)

### Comments
> Any additional comments that will aid with the review process.

If there's any other issues with this related to the new Dashboard then I can fix them when I get back.
